src/org/skypro/skyshop/product/FixPriceProduct.java

```diff
@@ -0,0 +1,26 @@
+package org.skypro.skyshop.product;
+
+public class FixPriceProduct extends Product{
+    public static int getPrice;
+    private final int FIXED_PRICE = 50;
+
+    public FixPriceProduct(String productName, int productPrice) {
+        super(productName, productPrice);
+    }
+
+    @Override
+    public boolean isSpecial() {
+        boolean isSpecial = true;
+        return isSpecial;
```

Comment:
какой смысл в переменной?
```suggestion
        return true;
```